pilot/pkg/serviceregistry/kube/controller/pod.go

```diff
@@ -134,7 +134,15 @@ func GetPodConditionFromList(conditions []v1.PodCondition, conditionType v1.PodC
 func (pc *PodCache) labelFilter(old, cur *v1.Pod) bool {
 	// If labels updated, trigger proxy push
 	if cur.Status.PodIP != "" && !reflect.DeepEqual(old.Labels, cur.Labels) {
-		pc.proxyUpdates(cur.Status.PodIP)
+		if len(cur.Status.PodIPs) > 0 {
+			var podIPs []string
+			for _, itemIP := range cur.Status.PodIPs {
```

Comment:
extract this into a function

Reply:
okay, extract function named `GetPodIPs`.